Keyboard/KeyboardViewController.swift

```diff
@@ -84,6 +84,10 @@ class KeyboardViewController: UIInputViewController, KeyboardDelegate {
         
         swipeLeftRecognizer.direction = .left
         swipeLeftRecognizer.addTarget(self, action: #selector(self.didSwipeLeft))
+        
+        spaceUtterance = createUtterance(word: "space")
```

Comment:
I believe audio settings could change without KeyboardViewController being re-initialized, so space/period/backspace utterances would not use the changed settings values.